roles/kubernetes/node/templates/kubelet.rkt.service.j2

```diff
@@ -62,7 +62,8 @@ ExecStart=/usr/bin/rkt run \
                 $KUBELET_ARGS \
                 $DOCKER_SOCKET \
                 $KUBELET_REGISTER_NODE \
-                $KUBELET_NETWORK_PLUGIN
+                $KUBELET_NETWORK_PLUGIN \
+                $KUBELET_CLOUDPROVIDER
```

Comment:
Got it, thanks! In this case I would refactor kubelet.service.j2 I would make one template instead of 2. That will help to prevent such issues in futures